types/theme-ui/tsconfig.json

```diff
@@ -14,7 +14,8 @@
             "../"
         ],
         "paths": {
-            "@styled-system/css": ["styled-system__css"]
+            "@styled-system/css": ["styled-system__css"],
+            "@emotion/serialize": ["emotion__serialize"]
```

Comment:
@callumacrae I have a feeling this shouldn't be here, since `@emotion/serialize` doesn't get installed in `@types`. 

Maybe try removing it? Or probably it should be `node_modules/@emotion/serialize/types`?

Reply:
Will see if that works!

Reply:
@callumacrae Can you please remove this and retry building? The pipeline was fixed today! 😃

Reply:
Done :)